### PR TITLE
Add the new findSkip overload to the change log

### PR DESCRIPTION
**Pull request by JackStouffer**
@wilzbach

### CODE REVIEW

File: changelog/std-algorithm-searching-findSkip.dd [hunk 1]
```diff
@@ -0,0 +1,12 @@
+findSkip can now skip elements using just a predicate function 
+
+Previously, $(REF findSkip, std, algorithm, searching) could only
+be used to find a specific string. Now, a new overload allows elements
+in a range to be skipped over if the passed function returns `true`:
+
+-------
+import std.ascii : isWhite;
+string s = "    abc";
+assert(findSkip!isWhite(s) == 2 && s == "abc");
```
Comment (ZombineDev) [inline changelog/std-algorithm-searching-findSkip.dd]: Here `findSkip!isWhite(s) ` should return `4`.

File: changelog/std-algorithm-searching-findSkip.dd [hunk 2]
```diff
@@ -0,0 +1,12 @@
+findSkip can now skip elements using just a predicate function
```
Comment (ZombineDev) [inline changelog/std-algorithm-searching-findSkip.dd]: Surround `findSkip` in backticks

**Comment by wilzbach** [comment]
Thanks a lot for doing this @JackStouffer!

**Comment by wilzbach** [comment]
> Here findSkip!isWhite(s) should return 4.

That was a tab. A replaced it with three spaces.

**Pull request merged**